src/heroes/doomfist/slam.opy

```diff
@@ -138,13 +140,17 @@ def startSlamAnimation():
     @Name "[doomfist/slam.opy]: startSlamAnimation()"
 
     # Start slam animation
+    if eventPlayer.slam_to_use == SlamType.GROUND:
+        eventPlayer.startForcingPosition(updateEveryTick(eventPlayer.getPosition()), false)
```

Comment:
what does this do

Reply:
the regular seismic slam will be able to slightly get its own distance for 1 tick, messing up the custom one, adding a startforcingposition for a tick will deny the vanilla impulse and let the custom one not get messed up